modules/solid_mechanics/test/tests/free_expansion_abs_ref/free_expansion_abs_ref.i

```diff
@@ -47,13 +47,13 @@
     use_displaced_mesh = false
   []
   [ref_x]
-    type = TagVectorAux
+    type = ReactionForceAux
     variable = 'ref_x'
     vector_tag = 'ref'
     v = 'disp_x'
   []
   [ref_y]
-    type = TagVectorAux
+    type = ReactionForceAux
```

Comment:
I dont think it is a force here? This is an absolute value vector tag for 'ref'

Reply:
This might have to be a `AbsoluteReactionForceAux` or something like that... @bwspenc can you comment?